src/main/resources/application-k8s-beta.yml

```diff
@@ -19,6 +19,8 @@ bb-fuel:
     transactions: http://transactions.${environment.name}.${environment.domain}/integration-api
     user: http://user-manager.${environment.name}.${environment.domain}/integration-api
     accountStatement: http://accountstatementsintegration.${environment.name}.${environment.domain}/service-api
+    pockets: http://edge.${environment.name}.${environment.domain}/api/pocket-tailor/client-api
+    pocketsArrangements: http://pocketsprovidermock.${environment.name}.${environment.domain}/service-api
```

Comment:
Here, you are trying to call arrangement-outbound integration service which is deployed as https://stash.backbase.com/projects/SDLC/repos/beck-helm/browse/helmfile.yaml#2221